src/main/java/com/truelayer/java/mandates/entities/GetConstraintsResponse.java

```diff
@@ -0,0 +1,20 @@
+package com.truelayer.java.mandates.entities;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+@Builder
+@Getter
+@ToString
+@EqualsAndHashCode
```

Comment:
Unless we build this somewhere (usually we don't for responses) you can simply use `@Value`, which includes all these (except the Builder)

Reply:
I'm using the builder in the acceptance test 😅